plutus-conformance/README.md

```diff
@@ -0,0 +1,63 @@
+# Plutus Core Conformance Test Suite
+
+**Note: This package is a work-in-progress.**
+
+This package aims to provide an official and comprehensive test suite that checks that the behaviour of Plutus conforms with the specified behaviour. We run all the tests in our CI to ensure continuous conformance. Users are encourage to contribute test cases to this collection of tests and utilize the suite (e.g., running the tests on alternative implementations of Plutus Core).
+
+## Specification
+
+The tests currently covers or will cover:
+
+- An evaluation test suite for UPLC (`uplc-eval-test`)
+- A typechecking test suite for TPLC, including checking of alpha equivalence. (`tplc-typecheck-test`)
+- An evaluation test suite for TPLC (`tplc-eval-test`)
+- A TPLC to UPLC erasure test suite
+- Costing conformance 
+- Coverage test
+
+<!-- ## Testing alternative implementation
+
+We provide a function  -->
+
+## Untyped Plutus Core Evaluation
+
+The `uplc-eval-test` test suite ensures that the input untyped plutus core programs evaluate to the expected output. The expected output is obtained by evaluating the programs using the CEK machine.
```

Comment:
I think the point of this suite is to test against the expected _semantics_ of PLC. So the fact that we use the CEK machine in our implementation of that is irrelevant!

Reply:
Well, but we *won't* be testing them against the CEK machine, we'll be testing them against the *checked-in input*, which for all they know we could have derived by hand!

This will be different if/when we start providing tests where we generate inputs and evaluate them with our implementation and theirs. Then we really will be testing against the CEK machine.

Anyway, you can leave it in for now if you like!

Reply:
I guess what I was trying to say was that the checked-in input was gotten from the CEK machine **at some point**. I don't want to be misleading so I'll comment that out for now.